index.js

```diff
@@ -36,7 +38,20 @@ function getCartTotal(cart) {
   - Any of the products in the `products` array does not have a `priceInCents` key.
 */
 function filterProductsByPriceRange(products, min, max) {
+
+  if(!products.length ||
+    typeof min !== "number" ||
+    typeof max !== "number"||
+    max === 0 ||
+    min > max ||
+    min < 0 ||
+    max < 0)
+
+     throw "error";
```

Comment:
in the future you should include a message with what exactly is going wrong. It'll help you debug as a developer or let the user know what information is incorrect/missing